junit-platform-engine/src/main/java/org/junit/platform/engine/reporting/ReportEntry.java

```diff
@@ -73,8 +73,8 @@ public static ReportEntry from(String key, String value) {
 	}
 
 	private void add(String key, String value) {
-		Preconditions.notBlank(key, "key must not be null or blank");
-		Preconditions.notBlank(value, "value must not be null or blank");
+		Preconditions.notBlank(key, "Key must not be null or blank");
```

Comment:
This message seems to refer to parameter, and there is no parameter `Key` now.
It is not consistent with line 55 as well.

Similar for line below.

Reply:
Makes sense, pushed a fix to revert this